### DISCUSSION TITLE
write quit shortcut?

### DISCUSSION THREAD

**Discussion by matthiasdebernardini** [helix-editor/helix#4112] [Q&A] [answered]
Sorry if this is spam, I tried joining the matrix room but my Element matrix client wouldn't allow me (not sure what I am doing wrong).

I like using "ZZ" (Shift + z + z) to quit vim.

tried remapping 
`ZZ = ":write-quit"`

but that gave an error

Is this remapping possible? How would I do it?

What is the helix way to quickly and easily write and quit? 

Thank you

**Comment by carrascomj** [comment, marked as answer]
Very close, I have it working like this
```toml
[keys.normal]
Z = {Q = ":quit!", Z = ":write-quit!"}
```

in my `.config/helix/config.toml`.

**Reply by matthiasdebernardini** [reply]
hmm, this doesn't work, it pops up an info window 

when you remap a key, does the old mapping get deleted?

**Reply by matthiasdebernardini** [reply]
ok so it looks like I have to fully close the editor and reopen it for the config to take effect, however, the info window still opens up when I press "ZZ" doesn't seem like its a problem though

**Comment by matthiasdebernardini** [comment]
perhaps an automatic save based on minimum number of actions or edits would also be a good addition?
